test/screenshot/infra/lib/cbt-api.js

```diff
@@ -400,24 +401,39 @@ https://crossbrowsertesting.com/account
    */
   async killZombieSeleniumTests() {
     // NOTE: This only returns Selenium tests running on the authenticated CBT user's account.
-    // It does NOT return Selenium tests running under other users.
+    // It does NOT return Selenium tests running under other CBT user accounts.
     /** @type {!CbtSeleniumListResponse} */
     const listResponse = await this.sendRequest_(
       getStackTrace('killZombieSeleniumTests'),
       'GET', '/selenium?active=true&num=100'
     );
 
-    const activeSeleniumTestIds = listResponse.selenium.map((test) => test.selenium_session_id);
+    if (!listResponse.selenium) {
+      this.warnMalformedResponse_(listResponse, 'list');
+      return;
+    }
+
+    this.logSeleniumSessions_(listResponse.selenium, 'Active Selenium sessions');
+
+    const activeSeleniumTestIds = listResponse.selenium.map((session) => session.selenium_test_id);
 
     /** @type {!Array<!CbtSeleniumInfoResponse>} */
     const infoResponses = await Promise.all(activeSeleniumTestIds.map((seleniumTestId) => {
       const infoStackTrace = getStackTrace('killZombieSeleniumTests');
       return this.sendRequest_(infoStackTrace, 'GET', `/selenium/${seleniumTestId}`);
     }));
 
-    const stalledSeleniumSessionIds = [];
+    this.logSeleniumSessions_(infoResponses, 'Selenium sessions info');
```

Comment:
I see two calls to `logSeleniumSessions_` (here and above). IIUC this one only logs detected active sessions. What does the previous one log? Do we need both of these or is this a lot of added noise? Do the responses contain anything sensitive we should avoid outputting?

Reply:
Annoyingly, the "list all active Selenium sessions" endpoint returns everything we need _except_ the list of commands.

So I have to loop through the active sessions, pull out their test IDs, and query each session individually to get the commands.

I updated the PR description with a screenshot of an example log message. As you can see, they're nearly identical, but I thought it would be useful to print both in case something goes wrong and we need to debug. It does add some noise, but at this point I'd rather have more information than less, so that we can track down future issues more easily.